lib/screens/wallet/components/tokens_cards/interactor/viewmodels/token_balances_bloc.dart

```diff
@@ -26,6 +26,8 @@ class TokenBalancesBloc extends Bloc<TokenBalancesEvent, TokenBalancesState> {
         add(const OnLoadTokenBalances());
       } else if (event is OnFiatCurrencyChangedEventBus) {
         add(const OnFiatCurrencyChanged());
```

Comment:
I actually think refresh and reload balance is broken on Seeds LW as well.

Reply:
Because it wasn't working and I didn't see any code in the LW that would make it work there.